CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+cmake_minimum_required(VERSION 3.5.0)
+project(cmake_common_scripts VERSION 0.1.0 LANGUAGES CXX)
```

Comment:
Does this project contain any C++?

If not: remove the `LANGUAGES CXX` part.

Reply:
Done.